feature/gribi/otg_tests/gribi_static_route_on_lemmings_test/README.md

```diff
@@ -0,0 +1,48 @@
+# GRIBI static route on lemmings test
```

Comment:
Looks like this has more coverage as well...

http://shortn/_FcU1Zv7i6k

If Lemmings SKU is already covered in existing tests , then we can use the same i guess.

Reply:
Purpose of this test case is to give minimum environment for developing the control plane by using the lemmings